homeassistant/components/remote/__init__.py

```diff
@@ -38,8 +41,12 @@
 
 SCAN_INTERVAL = timedelta(seconds=30)
 SERVICE_SEND_COMMAND = 'send_command'
+SERVICE_SEND_COMMANDS = 'send_commands'
 SERVICE_SYNC = 'sync'
 
+DEFAULT_NUM_REPEATS = '1'
+DEFAULT_DELAY_SECS = '0.4'
```

Comment:
@everix1992 Awesome! Thanks a lot man. I'll test this PR on my personal Harmony Hub as well this week.